cmd/crc/cmd/console.go

```diff
@@ -53,6 +55,12 @@ func runConsole(arguments []string) {
 
 	if consoleURLMode {
 		output.Out(result.ClusterConfig.WebConsoleURL)
+	}
+	if consolePrintCredentials {
+		output.Out("To login as a normal user, username is 'developer' and password is 'developer'.")
+		output.OutF("To login as an admin, username is 'kubeadmin' and password is '%s'.", result.ClusterConfig.KubeAdminPass)
```

Comment:
It need a `\n` at the end otherwise this will look as following.

```
$ ./crc console --credentials
To login as a normal user, username is 'developer' and password is 'developer'.
To login as an admin, username is 'kubeadmin' and password is 'BMLkR-NjA28-v7exC-8bwAk'.✔ ~/work/github/crc/out/linux-amd64 [pr/626 L|✔] 
```

Reply:
`Out` IIRC adds the `\n` while `OutF` does not.

Reply:
Fixed in the upcoming iteration.